build/README.md

```diff
@@ -110,6 +110,10 @@ Table of Contents
       * [Error: cluster-admin-binding already exists](#error-cluster-admin-binding-already-exists)
       * [Error: releases do not exist](#error-releases-do-not-exist)
       * [I want to use pprof to profile the controller](#i-want-to-use-pprof-to-profile-the-controller)
+      * [Error: Kubernetes cluster unreachable](#error-kubernetes-cluster-unreachable-invalid-configuration-no-configuration-has-been-provided)
```

Comment:
Thanks for adding these. I noticed while checking the file that the headings for these new sections seem to be the wrong size (they use ### instead of #### like the ones above them in the same section). While you are fixing this file, could you adjust the headings for these sections at the bottom of the file?